vm/src/pyobject.rs

```diff
@@ -85,9 +84,6 @@ impl fmt::Display for PyObject<dyn PyObjectPayload> {
             }
         }
 
-        if let Some(PyModule { ref name, .. }) = self.payload::<PyModule>() {
```

Comment:
is there any code depending on here? it is not possible to keep compatibility here without name field.

Reply:
We should use payload as little as possible outside of the object itself, so this removal seems good to me!

Reply:
I confirmed current test code doesn't hit this line